azure_pipeline.yaml

```diff
@@ -104,6 +144,19 @@ stages:
           agentPool: ${{ variables.agentPool }}
           terraformVersion: ${{ variables.terraformVersion }}
 
+  - stage: ApplyCftAppsLBSandbox
+    displayName: "Apply CFT APPs LB - Sandbox"
```

Comment:
You don't need a display name as it is taken by the template.

Reply:
This section isn't in the template (stage)